app/assets/stylesheets/base.scss.erb

```diff
@@ -477,32 +468,34 @@ cursor: pointer;
 }
 
 .CardOnGraph .metacodeTitle {
-  font-style: italic;
-  font-family: 'vinyl';
-  text-transform: uppercase;
-  position: absolute;
+  font-family: 'din-regular';
   line-height: 24px;
   height: 26px;
-  font-size: 24px;
-  display: none;
-  width: 90%;
-  padding: 13px 0 9px 10%;
-  background-color: #E0E0E0;
+  font-size: 18px;
+  padding: 13px 24px 9px 24px;
   color: #424242;
+  width: 120px;
+  max-width: 120px;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  overflow: hidden;
 }
 .permission.canEdit .metacodeTitle {
   cursor:pointer;
 }
 
 .permission.canEdit .expandMetacodeSelect {
-  position: absolute;
-  top: 16px;
-  right: 16px;
+  position: relative;
+  top: 2px;
+  left: 4px;
   width: 16px;
   height: 16px;
   background-image: url(<%= asset_data_uri('arrowright_sprite.png') %>);
   background-repeat: no-repeat;
   background-position: 0 -32px;
+  display: inline-block;
+  -webkit-transform: rotate(90deg);
+  transform: rotate(90deg);
```

Comment:
remember my browser support PR? I know we're not there yet, but what set of browsers does these two transforms support? i'm assuming safari/chrome/opera are good, what about FF and Edge? I'm comfortable dropping IE11 if you want

This question is specific, but the answer depends on a more general question about our browser support strategy. And I guess our CSS support doesn't HAVE to match our JS, but it'd be nice to have a clear answer on how far to go to support different browsers for this sort of thing. 

I hope this comment makes sense. I can clarify if not

Reply:
looks like, from 'caniuse' that I don't even need the webkit one actually.

Reply:
awesome